q2_feature_table/_examples.py

```diff
@@ -0,0 +1,357 @@
+
+import numpy as np
+from biom import Table
+
+import qiime2
+from qiime2 import Artifact
+
+
+rep_seqs_1_url = (f'https://data.qiime2.org/{qiime2.__release__}/'
+                  'tutorials/metadata/rep-seqs.qza')
+rep_seqs_2_url = (f'https://data.qiime2.org/{qiime2.__release__}/'
+                  'tutorials/phylogeny/rep-seqs.qza')
+taxonomy_1_url = ('https://docs.qiime2.org/jupyterbooks/cancer-microbiome-'
+                  'intervention-tutorial/data/030-tutorial-downstream/020-'
+                  'taxonomy/taxonomy.qza')
+moving_pics_ft_url = (f'https://data.qiime2.org/{qiime2.__release__}/'
+                      'tutorials/filtering/table.qza')
+moving_pics_md_url = (f'https://data.qiime2.org/{qiime2.__release__}/'
+                      'tutorials/moving-pictures/sample_metadata.tsv')
+
+
+def ft1_factory():
+    return Artifact.import_data(
+        'FeatureTable[Frequency]',
+        Table(np.array([[0, 1, 3], [1, 1, 2]]),
+              ['O1', 'O2'],
+              ['S1', 'S2', 'S3']))
+
+
+def ft2_factory():
+    return Artifact.import_data(
+        'FeatureTable[Frequency]',
+        Table(np.array([[0, 2, 6], [2, 2, 4]]),
+              ['O1', 'O3'],
+              ['S4', 'S5', 'S6']))
+
+
+def ft3_factory():
+    return Artifact.import_data(
+        'FeatureTable[Frequency]',
+        Table(np.array([[0, 4, 9], [4, 4, 8]]),
+              ['O1', 'O4'],
+              ['S7', 'S8', 'S9']))
+
+
+def feature_table_merge_example(use):
+    feature_table1 = use.init_artifact('feature_table1', ft1_factory)
+    feature_table2 = use.init_artifact('feature_table2', ft2_factory)
+
+    merged_table, = use.action(
+        use.UsageAction(plugin_id='feature_table',
+                        action_id='merge'),
+        use.UsageInputs(tables=[feature_table1, feature_table2]),
+        use.UsageOutputNames(merged_table='merged_table'),
+    )
+
+
+def feature_table_merge_three_tables_example(use):
+    feature_table1 = use.init_artifact('feature_table1', ft1_factory)
+    feature_table2 = use.init_artifact('feature_table2', ft2_factory)
+    feature_table3 = use.init_artifact('feature_table3', ft3_factory)
+
+    merged_table, = use.action(
+        use.UsageAction(plugin_id='feature_table',
+                        action_id='merge'),
+        use.UsageInputs(
+            tables=[feature_table1, feature_table2, feature_table3],
+            overlap_method='sum'
+        ),
+        use.UsageOutputNames(merged_table='merged_table'),
+    )
+
+
+def feature_table_merge_seqs(use):
+    seqs1 = use.init_artifact_from_url('seqs1', rep_seqs_1_url)
+    seqs2 = \
+        use.init_artifact_from_url('seqs2', rep_seqs_2_url)
+
+    merged_data, = use.action(
+        use.UsageAction('feature_table', 'merge_seqs'),
+        use.UsageInputs(
+            data=[seqs1, seqs2]
+        ),
+        use.UsageOutputNames(
+            merged_data='merged_data'
+        )
+    )
+
+
+def feature_table_filter_seqs_metadata1(use):
```

Comment:
```suggestion
def feature_table_filter_seqs_subject1(use):
```
I'd also change this name (and the following test as well) to indicate the different ways you're filtering. I was initially wondering if you were using two different metadata files for your filtering.